### ISSUE TITLE
Add aria-label to pagination container

### ISSUE THREAD

**Issue by paperplanefactory** [lesterchan/wp-pagenavi#72]
The container that manages pagination elements should be accompanied by an aria-label for better accessibility, for example:

```
<div class="wp-pagenavi" role="navigation" aria-label="Pagination">
</div>
```
